GSA/openacr
# Electronic documents, chapter shortname, pa11y-ci update

A lot of changes, here is a summary:

1. Renaming 'Electronic Docs' to 'Electronic Documents' for #280 
2. Added `short_label` to the chapter object for the catalog schema so we don't have to have a separate file in the editor for those values.
3. Updated pa11y-ci to latest version so it running more up to date axe-core checks. Did have to ignore contract checks because it cannot calculate the contrast of the table header.
4. With the package changes I updated the license file. See https://github.com/GSA/openacr/tree/main/license for details.

**QA**

1. Review the HTML and Markdown files to confirm the rename of 'Electronic Docs' to 'Electronic Documents'. You can download generated HTML from the GitHub action: https://github.com/GSA/openacr/actions/runs/1725653905
2. Short label will be tested in the editor.
3. Confirm the a11y-tests run without errors.
